src/components/modules/paste.ts

```diff
@@ -306,31 +313,66 @@ export default class Paste extends Module {
     }
   }
 
+  /**
+   * Get tags from tags configuration.
+   *
+   * @param {string | object} tagsConfig - tag name or sanitize config object.
+   * @returns {string[]} array of tags.
+   */
+  private getTags = (tagsConfig: (string | object)): string[] => {
+    /**
+     * If tagsConfig is string, return it as an array.
+     */
+    if (_.isString(tagsConfig)) {
+      return [ tagsConfig.toUpperCase() ];
```

Comment:
I think, the uppercasing is out of the scope of this method and should be removed from it to somewhere else